src/index.html

```diff
@@ -22,9 +22,7 @@ <h1 class="title head-title">Leader Board</h1>
                   <div class="columns score">
                     <div class="column"><p class="title">Recent Scores</p></div>
                     <div class="column">
-                      <button class="button is-success" type="button">
-                        Refresh
-                      </button>
+                      <button class="button is-success" type="button" id="refresh">Refresh</button>
```

Comment:
- [ ] The refresh button should hit the `GET API` endpoint causing the scores list to change but it seems not to be working. Please check the API functions and make sure they work properly.